packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -610,7 +610,7 @@ export class PeerManager {
     try {
       await this.libp2p.hangUp(peer);
     } catch (e) {
-      this.logger.warn("Unclean disconnect", {peer: prettyPrintPeerId(peer)}, e as Error);
+      this.logger.debug("Unclean disconnect", {peer: prettyPrintPeerId(peer)}, e as Error);
```

Comment:
Reference: https://github.com/ChainSafe/lodestar/issues/5359#issuecomment-1519129570 by @nflaig for moving from `warn` to `debug`.

Reply:
Definitely must be debug level

Reply:
To be fixed by
- https://github.com/ChainSafe/lodestar/pull/5541
We might wanna include that in 1.9 as there have been concerns from several users in regard to this log